addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1041,7 +1041,7 @@ export class Orderline extends PosModel {
         var taxes_ids = this.tax_ids || product.taxes_id;
         taxes_ids = taxes_ids.filter((t) => t in this.pos.taxes_by_id);
         var taxdetail = {};
-        var product_taxes = this.pos.get_taxes_after_fp(taxes_ids, this.order.fiscal_position);
+        var product_taxes = this.pos.get_taxes_after_fp(taxes_ids, this.order?.fiscal_position);
```

Comment:
@adgu-odoo I had to modify this because order was always null when running the tour. I was not able to reproduce the issue outside of the tour. But I think that it's happening because the `product-reminder` is still there when we delete the line